locals.tf

```diff
@@ -10,7 +15,7 @@ locals {
     name                       = local.name
     description                = "A Helm chart for portworx"
     chart                      = "portworx"
-    repository                 = "https://raw.githubusercontent.com/portworx/helm/eks-blueprint/repo/staging"
+    repository                 = "https://raw.githubusercontent.com/portworx/eks-blueprint-helm/main/repo/stable"
     version                    = "2.11.0"
```

Comment:
remove version from here, let the helm version be set. Put this in the readme if the user wants to change it.

Reply:
I think this is a required field because we can have multiple release right, so this version is used to pick one package by cross referring index.yaml

Reply:
Is this version for the helm chart? If yes then that makes sense.